packages/@react-spectrum/menu/docs/ActionMenu.mdx

```diff
@@ -0,0 +1,259 @@
+## Events
+ActionMenu supports the `onAction` callback which receives the selected item. See [Selection](#selection) for more information.
+
+```tsx example
+function Example() {
+  let [action, setAction] = React.useState('');
+
+  return (
+    <>
+      <ActionMenu onAction={setAction}>
+        <Item key="cut">Cut</Item>
+        <Item key="copy">Copy</Item>
+        <Item key="paste">Paste</Item>
+      </ActionMenu>
+      <p>Action: {action}</p>
+    </>
+  );
+}
+```
+
+## Sections
+ActionMenu supports sections in order to group options. Sections can be used by wrapping groups of Items in a `Section` component. Each `Section` takes a `title` and `key` prop.
+
+### Static Items
+
+```tsx example
+<ActionMenu>
+  <Section title="Styles">
+    <Item key="bold">Bold</Item>
+    <Item key="underline">Underline</Item>
+  </Section>
+  <Section title="Align">
+    <Item key="left">Left</Item>
+    <Item key="middle">Middle</Item>
+    <Item key="right">Right</Item>
+  </Section>
+</ActionMenu>
+```
+
+### Dynamic Items
+
+Sections used with dynamic items are populated from a hierarchical data structure. Similarly to the props on ActionMenu, `<Section>` takes an array of data using the `items` prop.
+
+```tsx example
+function Example() {
+  let openWindows = [
+    {
+      name: 'Left Panel',
+      id: 'left',
+      children: [
+        {id: 1, name: 'Final Copy (1)'}
+      ]
+    },
+    {
+      name: 'Right Panel',
+      id: 'right',
+      children: [
+        {id: 2, name: 'index.ts'},
+        {id: 3, name: 'package.json'},
+        {id: 4, name: 'license.txt'}
+      ]
+    }
+  ];
+
+  return (
+    <ActionMenu
+      items={openWindows}>
+      {item => (
+        <Section items={item.children} title={item.name}>
+          {item => <Item>{item.name}</Item>}
+        </Section>
+      )}
+    </ActionMenu>
+  );
+}
+```
+
+### Accessibility
+
+Sections without a `title` must provide an `aria-label` for accessibility.
+
+## Complex Menu Items
+
+Items within ActionMenu also allow for additional content used to better communicate options. Icons and descriptions can be added to the `children` of `Item` as shown in the example below. If a description is added, the prop `slot="description"` must be used to distinguish the different `<Text>` elements.
+
+```tsx example
+import {Keyboard, Text} from '@react-spectrum/text';
+<ActionMenu>
+  <Item key="cut" textValue="cut">
+    <Cut size="S"/>
+    <Text>Cut</Text>
+    <Keyboard>⌘X</Keyboard>
+  </Item>
+  <Item key="copy" textValue="copy">
+    <Copy size="S"/>
+    <Text>Copy</Text>
+    <Keyboard>⌘C</Keyboard>
+  </Item>
+  <Item key="paste" textValue="paste">
+    <Paste size="S"/>
+    <Text>Paste</Text>
+    <Keyboard>⌘V</Keyboard>
+  </Item>
+</ActionMenu>
+```
+
+## Props
+
+<PropTable component={docs.exports.ActionMenu} links={docs.links} />
+
+## Visual options
+
+### Autofocus
+
+Applying `autoFocus` to the ActionMenu sets focus to the ActionMenu trigger.
+
+### Disabled items
```

Comment:
It's called "Disabled" in Menu. but that is because the menu only has items that can be disabled. Since this is an ActionMenu, we can disable items or the trigger so I was more specific.

Reply:
Should we add a `isDisabled` example as well then?

Reply:
Sure!

I also decided to add an Open example and found that we don't support `defaultOpen` or `onOpenChange`. Opening #2728 to fix that.